### DISCUSSION TITLE
能否移植 APP 首页中的不感兴趣到网页端

### DISCUSSION THREAD

**Discussion by Skylarzz** [the1812/Bilibili-Evolved#3664] [功能建议 / Ideas]
网页端直接拉黑似乎过于暴力

**Comment by the1812** [comment]
和 #850 类似, 我拒绝移植手机端的 API

**Reply by Skylarzz** [reply]
呜哇呜哇哇哇
![b603-kakmcxc5298262](https://user-images.githubusercontent.com/14090545/189945125-fefeab1f-ea4a-4acc-913c-95bec0bcc790.gif)
